aries-site/src/pages/templates/popover.mdx

```diff
@@ -0,0 +1,69 @@
+import { Example } from '../../layouts';
+import {
+  PopoverAnatomy,
+  PopoverSimpleExample,
+  PopoverInlineExample,
+} from '../../examples';
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+  code='https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/popover/PopoverSimpleExample.js'
+>
+  <PopoverSimpleExample />
+</Example>
+
+## Use cases
+### Providing inline contextual help
+
+A Popover can be triggered through an icon-only button to offer helpful contextual information in a non-intrusive manner.
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+  code='https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/popover/PopoverInlineExample.js'
+>
+  <PopoverInlineExample />
+</Example>
+
+## Anatomy
+
+<Example caption="Diagram illustrating the 4 parts of a Popover." plain>
+  <PopoverAnatomy />
+</Example>
+
+| Label | Region        | Purpose                              | Required | Notes                                      |
+| :---: | ------------- | ------------------------------------ | :------: | ------------------------------------------ |
+| **1** | **Title**     | Identifies the Popover.               | Optional |                                            |
+| **2** | **Close Button** | Closes the Popover.                | Required |                                            |
```

Comment:
I believe our convention is to use a center-aligned Checkmark icon with aria-label="required" for required regions.

Reply:
Ill double check

Reply:
I can add it will look like the following in Layer.
![Screenshot 2024-10-16 at 9 42 08 PM](https://github.com/user-attachments/assets/01eed140-b01f-4a5e-a494-5b85f06c1a2a)